diff -urN aws-crt-python-0.12.6-orig/setup.py aws-crt-python-0.12.6/setup.py
--- aws-crt-python-0.12.6-orig/setup.py	2022-11-04 13:55:33.961356037 +0100
+++ aws-crt-python-0.12.6/setup.py	2022-11-04 13:56:10.601483810 +0100
@@ -295,9 +295,6 @@
         extra_link_args += ['-framework', 'Security']
 
     else:  # unix
-        # linker will prefer shared libraries over static if it can find both.
-        # force linker to choose static variant by using using "-l:libcrypto.a" syntax instead of just "-lcrypto".
-        libraries = [':lib{}.a'.format(x) for x in libraries]
         libraries += ['rt']
 
         # hide the symbols from libcrypto.a
